samples/04_gis_analysts_data_scientists/export_video_using_arcgis_api_for_python.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>@guneetmutreja we need to change the title and name of this notebook so it talks about the scenario the user has, such as Visualize Shrinking Lakes using Video, and not as a technology tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>